codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/ServiceGenerator.java

```diff
@@ -222,9 +244,27 @@ private void generateClientInvokeOperation() {
 
             generateConstructStack();
             writer.write("options := c.options.Copy()");
+            writer.write("");
```

Comment:
Minor nit, if this newline isn't there wouldn't have as many files/modules changed in generated code.

Reply:
Will update